.github/workflows/ci.yml

```diff
@@ -67,7 +67,7 @@ jobs:
         # Remove the container.
         docker rm ntr
         # Write the expected output (which does not contain a newline).
-        printf '{"version":2,"status":"pass","tests":[{"name":"say hi!","status":"pass","output":""}]}' \
+        printf '{"version":2,"status":"pass","tests":[{"name":"say hi!","status":"pass","output":"","test_code":"check hello() == \"Hello, World!\""}]}' \
```

Comment:
Locally, the slash is escaped, ie `\\"Hello, World!\\"`

@ee7 Any idea?

Reply:
After this issue, I think it's good other than the error messages need macro magic to fix line numbers and stuff